tensorflow/core/kernels/mkl/mkl_conv_ops.h

```diff
@@ -641,6 +641,9 @@ class MklConvBackpropCommonOp : public OpKernel {
     OP_REQUIRES(context, FormatFromString(data_format_str, &data_format_),
                 errors::InvalidArgument("Invalid data format"));
     OP_REQUIRES_OK(context, context->GetAttr("strides", &strides_));
+    if (strides_.size() < 4) {
+        return errors::InvalidArgument("strides dimensions must not be < 4. ");
```

Comment:
It seems we no longer need to use TF wrappers around errors:

> Use absl::AbortedError etc. instead of tsl::errors::Aborted. You will need to write explicitly the e.g. absl::StrCat. This is required to get correct absl::SourceLocation propagation within Google.